project/pyproject.toml.jinja

```diff
@@ -172,9 +172,14 @@ build_command = "pip install poetry && poetry build"
 
 [tool.semantic_release.changelog]
 exclude_commit_patterns = [
-  "chore.*",
-  "ci.*",
-  "Merge pull request .*",
+  '''chore(?:\([^)]*?\))?: .+''',
+  '''ci(?:\([^)]*?\))?: .+''',
+  '''refactor(?:\([^)]*?\))?: .+''',
+  '''style(?:\([^)]*?\))?: .+''',
+  '''test(?:\([^)]*?\))?: .+''',
+  '''build\((?!deps\): .+)''',
+  '''Merged? .*''',
+  '''Initial [Cc]ommit.*''', # codespell:ignore
```

Comment:
Question: why is it not the default in PSR? Seems like these aren't worth metnioning in the changelog...

Reply:
Prior decision before me (became maintainer for v9+) to only have that config default as an empty list. 

I have not added it because of time and priority (and no issue from the community requesting it). 

When I have considered setting it, essentially we have more than one parser type and I would need to dynamically set the default values per type. And at the time, I'm not sure folks would agreed with me as that is the default they wanted, so I left it as configurable.

Reply:
Got it, makes sense to me. It's definitely harder to do it for the library, generically than here.